paybybank-us/src/main/java/com/adyen/checkout/paybybankus/PayByBankUSConfiguration.kt

```diff
@@ -8,6 +8,121 @@
+    override val shopperLocale: Locale?,
+    override val environment: Environment,
+    override val clientKey: String,
+    override val analyticsConfiguration: AnalyticsConfiguration?,
+    override val amount: Amount?,
+    internal val genericActionConfiguration: GenericActionConfiguration,
+) : Configuration {
+
+    class Builder : ActionHandlingPaymentMethodConfigurationBuilder<PayByBankUSConfiguration, Builder> {
+
+        /**
+         * Initialize a configuration builder with the required fields.
+         *
+         * The shopper locale will match the value passed to the API with the sessions flow, or the primary user locale
+         * on the device otherwise. Check out the
+         * [Sessions API documentation](https://docs.adyen.com/api-explorer/Checkout/latest/post/sessions) on how to set
+         * this value.
+         *
+         * @param environment The [Environment] to be used for internal network calls from the SDK to Adyen.
+         * @param clientKey Your Client Key used for internal network calls from the SDK to Adyen.
+         */
+        constructor(environment: Environment, clientKey: String) : super(
+            environment,
+            clientKey,
+        )
+
+        /**
+         * Alternative constructor that uses the [context] to fetch the user locale and use it as a shopper locale.
+         *
+         * @param context A context
+         * @param environment The [Environment] to be used for internal network calls from the SDK to Adyen.
+         * @param clientKey Your Client Key used for internal network calls from the SDK to Adyen.
+         */
+        @Deprecated("You can omit the context parameter")
+        constructor(context: Context, environment: Environment, clientKey: String) : super(
+            context,
+            environment,
+            clientKey,
+        )
+
+        /**
+         * Initialize a configuration builder with the required fields and a shopper locale.
+         *
+         * @param shopperLocale The [Locale] of the shopper.
+         * @param environment The [Environment] to be used for internal network calls from the SDK to Adyen.
+         * @param clientKey Your Client Key used for internal network calls from the SDK to Adyen.
+         */
+        constructor(
+            shopperLocale: Locale,
+            environment: Environment,
+            clientKey: String
+        ) : super(shopperLocale, environment, clientKey)
+
+        override fun buildInternal(): PayByBankUSConfiguration {
+            return PayByBankUSConfiguration(
+                shopperLocale = shopperLocale,
+                environment = environment,
+                clientKey = clientKey,
+                analyticsConfiguration = analyticsConfiguration,
+                amount = amount,
+                genericActionConfiguration = genericActionConfigurationBuilder.build(),
+            )
+        }
+    }
+}
+
+fun CheckoutConfiguration.payByBankUS(
```

Comment:
Can you add tests for these methods as well?  Take a look at `PayByBankConfigurationTest`, it should be pretty much what you need